INSTALL.md

```diff
@@ -0,0 +1,85 @@
+# BUILDING FROM SOURCE
+
+By contributing to this project you agree to the [Code of Conduct](https://coralproject.net/code-of-conduct.html).
+
+## Requirements
+
+* [Node 7.2.1](https://nodejs.org/es/download/package-manager)
```

Comment:
Node version is specified in the `package.json` already, no need to duplicate it here.